jaydebeapi/__init__.py

```diff
@@ -367,12 +370,14 @@ def TimestampFromTicks(ticks):
     return apply(Timestamp, time.localtime(ticks)[:6])
 
 # DB-API 2.0 Module Interface connect constructor
-def connect(jclassname, url, driver_args=None, jars=None, libs=None):
+def connect(jclassname, url, driver_args=None, java_opts=None, jars=None, libs=None):
```

Comment:
Can you please add the java_opt at the end of the args. Otherwise this change will break backwards compatibility for code that doesn't use kwargs but relies on the order of the arguments.